recipe/conda_build_config.yaml

```diff
@@ -167,7 +173,7 @@ pixman:
 poppler:
   - 0.61
 proj4:
-  - 4
+  - 4.9.3
```

Comment:
Has changed SONAME in patch releases as noted in the [ABI/API report]( https://abi-laboratory.pro/tracker/timeline/proj/ ). Seems good to keep this locked down pretty tight.